File: 3123/3123.cs
public class Solution {
    public int FindEdgesInShortestPaths(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FindEdgesInShortestPaths(root.left), FindEdgesInShortestPaths(root.right));
    }
}

File: 3123/3123.go
func findEdgesInShortestPaths(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findEdgesInShortestPaths(root.Left)
	r := findEdgesInShortestPaths(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}